tool_manager.py

```diff
@@ -0,0 +1,83 @@
+import os
+import requests
+import zipfile
+import json
+
+# def parse_github_url(github_url):
```

Comment:
remove these comments if unused.

Reply:
You good sorry if you feel like I took yo long I got to much on my plate you did good, real good.